pnpm-workspace.yaml

```diff
@@ -0,0 +1,4 @@
+packages:
+  # https://github.com/changesets/changesets/issues/1133#issuecomment-1594844204
+  - .
```

Comment:
This is really neat, didn't realise we could accomplish this without displacing all the existing `skuba` code!